lib/kramdown/converter/math_engine/mathjaxnode.rb

```diff
@@ -28,29 +29,48 @@ module MathjaxNode
         npm = %x{npm --depth=1 list mathjax-node 2>&1}
       end
 
-      T2MPATH = File.join(npm.lines.first.strip, "node_modules/mathjax-node/bin/tex2mml")
-      /mathjax-node@/ === npm.lines.drop(1).join("\n") && File.exist?(T2MPATH)
+      MML_BIN = File.join(npm.lines.first.strip, "node_modules/mathjax-node/bin/tex2mml")
+      SVG_BIN = File.join(npm.lines.first.strip, "node_modules/mathjax-node/bin/tex2svg")
+
+      /mathjax-node@/ === npm.lines.drop(1).join("\n") && File.exist?(MML_BIN) && File.exist?(SVG_BIN)
     rescue
       false
     end
 
     def self.call(converter, el, opts)
       type = el.options[:category]
 
-      cmd = [T2MPATH]
+      format = converter.options[:math_engine_opts][:format].to_s
+      format = VALID_FORMATS.include?(format) ? format : 'mml'
+
+      cmd = []
+      cmd << const_get("#{format.upcase}_BIN")
       cmd << "--inline" unless type == :block
-      cmd << "--semantics" if converter.options[:math_engine_opts][:semantics] == true
-      cmd << "--notexhints" if converter.options[:math_engine_opts][:texhints] == false
+
+      if format == 'mml'
+        cmd << "--semantics" if converter.options[:math_engine_opts][:semantics] == true
+        cmd << "--notexhints" if converter.options[:math_engine_opts][:texhints] == false
+      end
+
       result = IO.popen(cmd << el.value).read.strip
 
-      attr = el.attr.dup
-      attr.delete('xmlns')
-      attr.delete('display')
-      result.insert("<math".length, converter.html_attributes(attr))
+      if format == 'mml'
+        attr = el.attr.dup
+        attr.delete('xmlns')
+        attr.delete('display')
+        result.insert("<math".length, converter.html_attributes(attr))
+      end
+
+      if format == 'svg'
+        if type == :block
+          result.insert(0, '<p>')
```

Comment:
Why are you wrapping a paragraph about the image in block mode? Wouldn't this make this an inline image? Better use a `<div>`, no?